templates/index.html

```diff
@@ -2,31 +2,52 @@
 <html>
 <head>
     <title>Notebook</title>
-
     <link rel="stylesheet" type="text/css" href="/static/css/stylesheet.css">
+
 </head>
 <body>
     <h2>Make Notes Simply! <br>
         Simply Make Notes!
     </h2>
-    <form action="/notes" method="post">
-        <p>Lable Your Note:</p><input id="title_for_new_note" type="text" placeholder="Note Title" name="note_title">
-        <br>
-        <p>Create Your Note:</p><input id="new_note_text_box" type="text" placeholder="Note" name="new_note">
-        <br>
-        <br>
-        <button id="new_note_button">Submit</button>
-    </form>
 
-    <h3>Your Notebook Log:</h3>
-    <div class="notebook_border">
+    <div id="status">
+    </div>
 
-        {% if not notes %}
+    <div id="fb-root"></div>
+        <script>
+    
+            (function(d, s, id) {
+                  var js, fjs = d.getElementsByTagName(s)[0];
+                  if (d.getElementById(id)) return;
+                  js = d.createElement(s); js.id = id;
+                  js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8&appId=1899079457005459";
+                  fjs.parentNode.insertBefore(js, fjs);
+                }(document, 'script', 'facebook-jssdk'));
+            
+        </script>
```

Comment:
can you move this script to the bottom of the page?

also ideally it'd be moved into it's own template so including it here would be a one-liner.